### PR TITLE
IVI-665 - Updating order sdk protos to match IVI protos and responses.

### PR DESCRIPTION
**Pull request by kwang-mythical**

### CODE REVIEW

**Review by keith-miller — APPROVED** [review]
lgtm

**Comment by keith-miller** [comment]
Let's not push this in until the SDK gets deployed to QA.

File: api/order/definition.proto [hunk 1]
```diff
@@ -9,10 +9,11 @@ import "google/protobuf/struct.proto";
 import "common/order/definition.proto";
 import "common/item/definition.proto";
 import "common/common.proto";
+import "google/protobuf/timestamp.proto";
 
 // Order definitions
-message PurchasedItems {
-  repeated PurchasedItem purchased_items = 1;
+message IssuedItems {
+  repeated IssuedItem item_types = 1;
```
Comment (keith-miller) [inline api/order/definition.proto:16]: This should stay called `purchased_items`. Calling it `item_types` is confusing.

File: api/order/definition.proto [hunk 2]
```diff
@@ -23,7 +24,7 @@ message Order {
   string total = 5;
   Address address = 6;
   oneof line_items {
-      PurchasedItems purchased_items = 7;
+      IssuedItems item_types = 7;
```
Comment (keith-miller) [inline api/order/definition.proto:27]: Same as above.

File: api/order/definition.proto [hunk 3]
```diff
@@ -54,23 +55,39 @@ message PurchasedItem {
...
+  google.protobuf.Struct metadata = 4;
+  string created_by = 5;
+  string request_ip = 6;
+  string environment_id = 7;
+  PaymentProviderId payment_provider_id = 8;
+  OrderRequestProviderProto payment_provider_data = 9;
   oneof line_items {
-    PurchasedItems purchased_items = 6;
-    string listing_id = 7;
+    IssuedItems item_types = 10;
```
Comment (keith-miller) [inline api/order/definition.proto:68]: Same as above.

File: api/order/definition.proto [hunk 4]
```diff
@@ -44,7 +45,7 @@ message Address {
   string country_iso_alpha_2 = 9;
 }
 
-message PurchasedItem {
+message IssuedItem {
   string game_inventory_id = 1;
   string item_name = 2;
   string item_type_id = 3;
```
Comment (keith-miller) [inline api/order/definition.proto:51]: `item_type_id` needs to be `game_item_type_id` now.

**Review by keith-miller — CHANGES_REQUESTED** [review]

File: api/order/definition.proto (hunk 1 above)
Comment (kwang-mythical) [reply api/order/definition.proto:16]: roger, we're gonna need a rename on IVI then

File: api/order/definition.proto (hunk 4 above)
Comment (kwang-mythical) [reply api/order/definition.proto]: yeah, your changes should've covered this, rebasing branch for update.

**Review by keith-miller — APPROVED** [review]

**Pull request merged**